Header header
float64 confidence
float64 priority
Flags flags
Shape2D shape
Transform2D transform
uuid_msgs/UniqueID uuid
string[] sensor_id
Motion2D motion
TrackingInfo tracking_info
string type_name
TypeCar type_car
TypeLanemarking type_lanemarking
TypeTrafficLight type_traffic_light
TypeStopMark type_stop_mark
